custom_components/smartthinq_sensors/wideq/device.py

```diff
@@ -65,7 +65,7 @@ class DeviceType(enum.Enum):
     DISHWASHER = 204
     TOWER_WASHER = 221
     TOWER_DRYER = 222
-    OVEN = 301
+    RANGE = 301
```

Comment:
Why rename? What exactly is a `range`? Just curiosity....

Reply:
In English, the word `range` is used to describe the combination of an `oven` and a `stove` into one kitchen appliance. http://www.rosedaleservice.com/the-difference-between-oven-stove-and-range/#:~:text=Basically%2C%20it%20is%20a%20sealed,enclosed%20device%2C%20not%20the%20stovetop.&text=Finally%2C%20range%20is%20used%20to,stove%20into%20once%20kitchen%20appliance.

Reply:
Fine, as a I said I was just curious, not my mother tongue...